test/glific_web/schema/sheet_test.exs

```diff
@@ -76,6 +83,29 @@ defmodule GlificWeb.Schema.SheetTest do
     assert get_in(query_data, [:data, "syncSheet", "sheet", "sheetDataCount"]) == 4
   end
 
+  test "create a sheet only sync partially due to errors in csv decode", %{manager: user} do
+    result =
+      auth_query_gql_by(:create, user,
+        variables: %{
```

Comment:
didn't get this testcase. Will we come across error then? Can you also check in testcase if notification is created coz that's the code u added when we'll come across error while syncing

Reply:
In this test, we only sync the first row, since from second row its invalid data, so we halt the process for this sheet.
Will add notification test too